src/pudl/transform/ferc1.py

```diff
@@ -6038,18 +4692,21 @@ def process_xbrl_metadata(self, xbrl_metadata_json) -> pd.DataFrame:
         metadata from the original column. We do this pre-processing before we
         call the main function in order for the calculation fixes and renaming to work
         as expected.
+
+        NOTE: Needs to happen before `process_xbrl_metadata_calculations`
         """
-        new_xbrl_metadata_json = xbrl_metadata_json
+        new_xbrl_metadata_json = self.xbrl_metadata_json
         # Get instant metadata
-        instant = pd.json_normalize(xbrl_metadata_json["instant"])
+        instant = pd.json_normalize(new_xbrl_metadata_json["instant"])
         # Duplicate instant metadata, and add starting/ending suffix
         instant = pd.concat([instant] * 2).reset_index(drop=True)
         instant["name"] = instant["name"] + ["_starting_balance", "_ending_balance"]
         # Return to JSON format in order to continue processing
         new_xbrl_metadata_json["instant"] = json.loads(
             instant.to_json(orient="records")
         )
-        tbl_meta = super().process_xbrl_metadata(new_xbrl_metadata_json)
+        self.xbrl_metadata_json = new_xbrl_metadata_json
+        tbl_meta = super().pre_process_xbrl_metadata
         return tbl_meta
```

Comment:
This is where the `electric_plant_depreciation_changes_ferc1` failure is happening!